cmd/run_service.go

```diff
@@ -107,7 +116,7 @@ func runWiretapService(wiretapConfig *shared.WiretapConfiguration, doc libopenap
 	bootedMessage(wiretapConfig)
 
 	// boot the http handler
-	handleHttpTraffic(wiretapConfig, wtService)
+	handleHttpTraffic(wiretapConfig, wtService, staticMockService)
```

Comment:
Use a message based struct here, just so we can expand it later without breaking anything.